### PR TITLE
MCXN94X CTIMER support

### PR DESCRIPTION
**Pull request by EmilioCBen**
Added CTimer Support to the MCXN94X Platform

### CODE REVIEW

**Review by decsny — DISMISSED** [review]

**Review by dleach02 — DISMISSED** [review]

**Review by mmahadevan108 — CHANGES_REQUESTED** [review]
Please add `counter` to the .yaml file.

**Comment by EmilioCBen** [comment]
> Please add `counter` to the .yaml file.

CTimer or Counter

File: boards/nxp/frdm_mcxn947/frdm_mcxn947_mcxn947_cpu0.yaml [hunk 1]
```diff
@@ -21,4 +21,5 @@ supported:
   - dac
   - i2c
   - watchdog
+  - ctimer
```
Comment (mmahadevan108) [inline boards/nxp/frdm_mcxn947/frdm_mcxn947_mcxn947_cpu0.yaml:24]: Please add `counter`

**Comment by mmahadevan108** [comment]
Please update the doc file to indicate this support

**Comment by EmilioCBen** [comment]
> Please update the doc file to indicate this support

 Thank you, updated.

**Review by decsny — DISMISSED** [review]

**Review by mmahadevan108 — DISMISSED** [review]

**Review by dleach02 — DISMISSED** [review]

**Review by decsny — APPROVED** [review]

**Review by mmahadevan108 — APPROVED** [review]

**Review by dleach02 — APPROVED** [review]

**Pull request merged**